ReSharper.FSharp/test/data/features/daemon/inferredTypeCodeVision/Common.fs.gold

```diff
@@ -0,0 +1,6 @@
+﻿module Module
+
+let |x|(0) = 1
+
+---------------------------------------------------------
+(0): CodeInsights: 
```

Comment:
@auduchinok Here I expect something like "(0): CodeInsights: **int**", but it's empty. I have no idea how to fix this.

Reply:
I'll look into it, thanks!